### ISSUE TITLE
Number of training samples used in paper

### ISSUE THREAD

**Issue by carlvinson70** [Cc-Hy/CMKD#43]
Hi there,

In the paper it says '~42k' raw KITTI samples are used - is this a typo? Apparently you only used ~18k train + eigen clean to get those results, or I am missing something here?

Hope you can clarify a bit. Thanks!

**Comment by Cc-Hy** [comment]
Hi, 
Sorry for being late, the best results in the paper use the entire kitti raw with ~42k samples, and the validation settings in this repo use ~18k train + eigen clean data. If you want to use the entire kitti raw, please use [this split file](https://github.com/Cc-Hy/CMKD/blob/main/data/kitti/ImageSets/kitti_raw.txt) and generate the label file for training.

**Comment by xiaoxusanheyi** [comment]
> 您好， 抱歉迟到了，本文中的最佳结果使用了整个 kitti raw 和 ~42k 样本，并且此存储库中的验证设置使用 ~18k 训练 + 特征干净数据。如果你想使用整个kitti raw，请使用[这个分割文件](https://github.com/Cc-Hy/CMKD/blob/main/data/kitti/ImageSets/kitti_raw.txt)并生成标签文件进行训练。

你好作者，这个实验的话 kitti-3D数据集使用那个trainval(~7K)训练的结果没有给出吗？
而表1中的KITTI-3D的测试结果是用~18k 训练 + 特征干净数据训练(如下图pkl文件) ，然后在测试集7518上进行测试得到的结果吗  
![image](https://github.com/Cc-Hy/CMKD/assets/113592382/af1fd273-4e89-4039-802a-9fe72f1037d7)
